src/parse/asp/builtins.go

```diff
@@ -987,11 +994,20 @@ func selectTarget(s *scope, l core.BuildLabel) *core.BuildTarget {
 	return subincludeTarget(s, l)
 }
 
+const (
```

Comment:
Could chuck these inside the `subrepo` function to be clear that's what they refer to (would also mean you wouldn't need to repeat `Subrepo` in each one)